http/http_server.hpp

```diff
@@ -101,26 +108,35 @@ class Server
 
     void startAsyncWaitForSignal()
     {
-        signals.async_wait(
-            [this](const boost::system::error_code& ec, int signalNo) {
-                if (ec)
+        signals.async_wait([this](const boost::system::error_code& ec,
+                                  int signalNo) {
+            if (ec)
+            {
+                BMCWEB_LOG_INFO("Error in signal handler{}", ec.message());
+            }
+            else
+            {
+                if (signalNo == SIGHUP)
+                {
+                    BMCWEB_LOG_INFO("Receivied reload signal");
+                    loadCertificate();
+                    startAsyncWaitForSignal();
+                }
+#ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
+                if (signalNo == SIGUSR1)
                 {
-                    BMCWEB_LOG_INFO("Error in signal handler{}", ec.message());
+                    BMCWEB_LOG_CRITICAL(
```

Comment:
Can we make this INFO since it is an info log?

Reply:
https://github.com/ibm-openbmc/bmcweb/commit/e18f1582ed53491ae83ff221ce79244008caf043 changed this log